x/ibc/04-channel/keeper/handshake.go

```diff
@@ -91,14 +91,12 @@ func (k Keeper) ChanOpenTry(
 		return types.ErrChannelExists(k.codespace, channelID)
 	}
 
-	/*
-		// TODO: Maybe not right
-		key := sdk.NewKVStoreKey(portID)
+	// TODO: Maybe not right
```

Comment:
@cwgoes is this correct to uncomment these lines? I thought this was related to the port work?